src/app/core/services/usage/usage.service.ts

```diff
@@ -0,0 +1,70 @@
+import { FirebaseAnalyticsService } from './firebase-analytics.service'
+import { Injectable } from '@angular/core'
+import { KafkaService } from '../kafka/kafka.service'
+import { SchemaType } from '../../../shared/models/kafka'
+import { UsageEventType } from '../../../shared/enums/events'
+import { WebIntent } from '@ionic-native/web-intent/ngx'
+
+@Injectable()
+export class UsageService {
```

Comment:
Promises in this service are not returned, e.g. the caller will not be able to know whether the call succeeded. This may be intentional (perhaps we don't have to wait until statistics are sent / crash if they fail). Then it would to add a comment to that effect, and perhaps with an annotation to ignore any warnings of missing promise returns, e.g. add
```
//noinspection JSIgnoredPromiseFromCall
```
to the class or to each of the ignored calls.

Reply:
Made the change.